### PR TITLE
docs, add vcf_cancer_sv to docs

### PR DESCRIPTION
**Pull request by hassanfa**
This PR updates doc with missing vcf_cancer_sv

**How to test**:
1. Read the docs!

**Expected outcome**:
To be able to see vcf_cancer_sv text 😅 

**Review:**
- [ ] code approved by
- [ ] tests executed by

### CODE REVIEW

**Comment by hassanfa** [comment]
I am not sure if any mortal being can review this absolutely massive PR, but you guys could give it a try.

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,8 @@ About changelog [here](https://keepachangelog.com/en/1.0.0/)
 ## [x.x.x]
 ### Added
 ### Fixed
+- missing `vcf_cancer_cv` to manual.
```
Comment (northwestwitch) [inline CHANGELOG.md:10]:
```suggestion
- missing `vcf_cancer_sv` to manual.
```

File: docs/admin-guide/load-config.md [hunk 2]
```diff
@@ -37,6 +37,7 @@ samples:
 vcf_snv: str(optional)
 vcf_sv: str(optional)
 vcf_cancer: str(optional)
+vcf_cancer_sv: str(optional)
```
Comment (northwestwitch) [inline docs/admin-guide/load-config.md:40]: 👍

**Review by northwestwitch — COMMENTED** [review]
I know that PR is already huge as it is, but since you are here you could also add vcf_cancer_sv_research?

**Review by northwestwitch — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (hassanfa) [reply CHANGELOG.md]: done!

**Comment by hassanfa** [comment]
Added research SVs as well.

**Comment by northwestwitch** [comment]
It's not passing the dev dependencies test, weird!

**Comment by dnil** [comment]
I get the impression that it's the freeze of `mkdocs-material` that has outlived it's practicality? Does anyone know why it's frozen? @moonso?

**Comment by moonso** [comment]
> I get the impression that it's the freeze of mkdocs-material that has outlived it's practicality? Does anyone know why it's frozen? @moonso?

No idea 🙄

**Pull request merged**